src/C2_economic_functions.py

```diff
@@ -31,12 +44,12 @@ def crf(project_life, wacc):
 def capex_from_investment(investment_t0, lifetime, project_life, wacc, tax):
     """
 
-    :param investment_t0:
-    :param lifetime:
-    :param project_life:
-    :param wacc:
-    :param tax:
-    :return:
+    :param investment_t0: first investment at the beginning of the project made at year 0
+    :param lifetime: time period over which investments and re-investments can occur. can be equal to or smaller than project_life
```

Comment:
```suggestion
    :param lifetime: time period over which investments and re-investments can occur. can be equal to, longer or shorter than project_life
```
Acutally, the lifetime can also be longer than the project life, especially for infrastructure (grid, ie. cabling...)